### DISCUSSION TITLE
About change color mode in mobile

### DISCUSSION THREAD

**Discussion by akazwz** [chakra-ui/chakra-ui#5389] [Ideas] [answered]
In the Mobile status bar, the light mode will be light and the dark mode will be black, but when changing color mode, the mobile status bar bg color won't change, how about updating the meta tag theme color at the same time?

**Comment by anubra266** [comment, marked as answer]
I don't think Chakra can handle this, you could do this yourself as it's going to be different in different environments.
E.g. in NextJS
```html
import Head from 'next/head';
import { useColorModeValue } from '@chakra-ui/react';

 <Head>
     <meta name="theme-color" content={useColorModeValue("#4285f4","red")}>
</Head>
```

**Reply by akazwz** [reply]
Thank you!
